css/style.css

```diff
@@ -1,14 +1,182 @@
 
  /* Use the 'Roboto' font from Google. Already added to your HTML header */
-
+* {
+    margin: 0;
+    padding: 0;
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    padding: 0;
+    margin: 0;
 }
 
+/* header style */
+.header{
+    display:flex;
+    width:100%;
+    padding: 10px 0;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+}
+.logo {
+    width: 35px;
+    height: 35px;
+    margin-top: 2%;
+}
+nav .navbar{
+    display: flex;
+    flex-direction: row;
+    padding: 3px;
+    color: rgb(148, 146, 145);
+    margin-top:5%;
+    text-transform: capitalize;
+}
+nav .navbar a{
+    margin-left: 25px;
+    text-decoration: none;
+}
+ .faded{
+   
+    color: rgb(148, 146, 145);
+}
+
+ a:hover{
+    color: rgb(70, 71, 70);
+}
+.current{
+    font-weight: bold;
+    color: rgb(70, 71, 70);
+}
+.current:hover{
+    font-weight: bold;
+    color: rgb(70, 71, 70);  
+}
+/* main body style*/ 
+main, .main{width: 100%;
+height: 100%;
+color: rgb(70, 71, 70);
+}
+.alignment{
+    text-align: center;
+}
+
+.sec1{
+    background-image: url("../img/first-background.jpg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    padding-top: 20%;
```

Comment:
the background image is being squashed on small screens. Try `background-size: cover` and background-position` if you need to move it around